### PR TITLE
Full webpage

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
### What?

I created a calculator UI using react.

### Why?
So I can use it to create a Single Page App.

### How?
- I installed create-react-app globally on my local machine
- I navigated to the folder in which I wanted to create the app on my local machine and used `create-react-app maths-magician` command to create the folder locally.
- I then installed linters.
- I pushed my code to a "Maths-Magician"repo I initially created.
- I created a components folder in the src folder.
- I created a calculator.js file in the components folder.
- I created the calculator UI using a function.
- I created the UI with the mobile first approach.
- I export the calculator from the Calculator.js and imported it into App.js
- I created a logic directory inside the src directory
- I created two files, calculate.js and operate.js inside the logic folder.
- I copied and pasted their respective files from Microverse course requirement.
- I installed Big
- I imported calculate into the calculator.js
- I declared properties inside the class
- I wrote an onClick function to handle events.
- I added the onClick to every button.
- Refactored the calculator.js and App.js modules to functions
- Changed the value when 'AC' button is clicked from null to 0.
- I install react router
- I created a pages folder inside components which contains the JavaScript and CSS files for the home and quotes pages.
- I routed the individual pages to their corresponding paths in App.js

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './navigation.css';
+// import Hamburger from 'hamburger-react';
```
Comment (Deepakdanger) [inline src/components/Navigation.js]: - [x] Kindly remove the commented code.

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                    <li className="menu-pop-up-li">Calculator</li>
+                    <li className="menu-pop-up-li">Quotes</li>
+                  </ul>
+                </div>
+              );
+            }
+            return null;
+          }}
+        />
+      </div> */}
```
Comment (Deepakdanger) [inline src/components/Navigation.js]: - [x] Kindly remove the commented code.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Haywayaheadshot , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
@Deepakdanger Thank you for reviewing my project. I have deleted all comments

**Review by victorokoroji — APPROVED** [review]
Hi @Haywayaheadshot, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅  
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
@vickymarz Thank you for reviewing my project.

**Pull request merged**